pkg/gvisor/enable.go

```diff
@@ -44,6 +44,7 @@ const (
 //   3. copies necessary containerd config files
 //   4. restarts containerd
 func Enable() error {
+	return fmt.Errorf("local image used correctly")
```

Comment:
I assume this was for debugging? I would have thought that Go would complain about unreachable code when compiling.

Reply:
Yup, I want to make sure minikube actually pulls the locally built gvisor image, so tests should fail with this error

Reply:
Currently it fails with compile error so you may want to remove the code after this line if you are intentionally returning an error.

Reply:
Where are you seeing the compile error? If I run `make out/gvisor-addon` locally the binary does compile, and running the docker image from `make gvisor-addon-image` gives me:

```
minikube$ docker run gcr.io/k8s-minikube/gvisor-addon:latest
2019/07/10 23:20:28 local image used correctly
```

Reply:
Hmm, I'm seeing the error in Travis. Go's compiler complains if code is unreachable.
https://travis-ci.org/kubernetes/minikube/builds/557049237#L541

Reply:
My bad. It was go vet the provides the error. In any case, the code afterwards isn't used.